examples/coap-server.c

```diff
@@ -921,10 +942,22 @@ fill_keystore(coap_context_t *ctx) {
       dtls_pki.sni_call_back_arg       = NULL;
     }
     if (!use_pem_buf) {
-      dtls_pki.pki_key.key_type = COAP_PKI_KEY_PEM;
-      dtls_pki.pki_key.key.pem.public_cert = cert_file;
-      dtls_pki.pki_key.key.pem.private_key = cert_file;
-      dtls_pki.pki_key.key.pem.ca_file = ca_file;
+      if ((key_file && strncasecmp (key_file, "pkcs11:", 7) == 0) ||
+          (cert_file && strncmp (cert_file, "pkcs11:", 7) == 0) ||
```

Comment:
I wonder if these two also have to be case-insentitive?

Reply:
Oops - yes - corrected.